src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  width: 200px;
+  margin: 0 auto;
+  padding-inline: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    display: flex;
+    padding-top: 40px;
+    padding-bottom: 4px;
+
+    .product-name {
+      margin: 0;
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 18px;
+      color: $color-primary;
+    }
+  }
+
+  &__product-code {
+    display: flex;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-secondary;
+    width: 200px;
+  }
```

Comment:
The BEM naming convention suggests using a single underscore for element names, so consider using '.card__product-code' instead of '.card__product-code'.